modules/build/src/main/scala/scala/build/input/Inputs.scala

```diff
@@ -313,8 +314,17 @@ object Inputs {
             if isShebangScript(String(content)) then
               s"$arg scripts with no file extension should be run with 'scala-cli shebang'"
             else
-              s"$arg: unrecognized source type (expected .scala or .sc extension, or a directory)"
-          else s"$arg: not found"
+              s"""$arg: unrecognized source type (expected .scala or .sc extension, or a directory),
+                 |if it's meant to be a script add a '!#' pointing to 'scala-cli shebang' in the top line
+                 |and run the source with 'scala-cli shebang $arg' or ${
```

Comment:
shouldn't explicitly use `scala-cli` here, we should switch to the actual command name that's been used... we probably need to pass `progName` from `cli`.
It's probably also safer to use `Shebang.names.headOption.map(_.mkString(" ")).getOrElse(Shebang.name)` instead of hardcoding `shebang` in the string 

So something like:
```suggestion
                 |and run the source with '$progName ${Shebang.name} $arg' or ${
```

Doesn't have to be fixed in this PR, as this applies to other strings in `Inputs.scala` as well, just noting